New Vox Ship Away

bigfunguss opened the pull request:
:cl: Funguss
maptweak: Replaces the old vox asteroid base with an overmap capable scavenger ship
tweak: Made a new ID card for vox scavengers, and added it to their outfit. Required to access the new ship's airlocks.
/:cl:

bigfunguss committed:
New Vox Ship Away

replaces old asteroid base for now

Albens commented:
This is obtusely armoured and defensible.
To an inane degree. 
The hull is exclusively made out of Torch Hull.
It maintains and effective thickness of 2-3 wide everywhere but a few areas. 

All the areas that are 1 thick are really defensible or really difficult to assault; 
Primary docking, canister storage, into the engine room with chokepoint.
Some of the 1 thick areas are just death to assault; 
OCP wall, Reactor, smelter. 

The default assault locations for Sec/Ascent/Skrell are even more defensible.
The Cockpit window is a chokepoint, hard to assault.
Both airlocks are awful chokepoints and hard to assault and easy to defend.
The Hydroponics window is a chokepoint that's hard to assault. 

Vox are Provocateurs; they _will_ be attacked by Skrell, Ascent, possibly others as time progresses.
They also maintain the chance of being assaulted by Torch-players.

This is cool.
But it might need to be toned back a bit to ensure we don't develop a meta of the Vox being unassailable.

lavillastrangiato commented:
why is it that the skrell ship has a ridiculous armory that could cheese the entire sector but a ship that's one cube thick is "ridiculously unassailable"?

Albens commented:
I believe this conversation was "held" in #pr-feedback. 
But if I must post it here. 

This ship is the single most likely vessel aside from the Torch(+T Shuttles) to get attacked
It is piloted by the single most antagonistic race (Ascent aren't actually that antagonistic, when you consider direct action)
The issues I pointed out are incredibly simple. 

<details>

 ![Image of](https://i.gyazo.com/4a234062476d1d0822d02d2c071ebde0.png)
Think about assaulting the arrows
And how it would take 1 vox to hold off up to 4+ attackers ez
Think of the red boxes; how those are death to assault
Think of the purple boxes, how those are still death to assault
</details>
Comparisons to the other Provacs aren't as relevant as one might expect. 
<details>
The Skrell can be attacked post-wakeup, just takes effort to get through Shields. 
They have a large ship, with windows, multiple entry points, expansive hallways and rooms. 

The Ascent aren't attackable at this time due to their lack of landing sites, but it's not unlikely they will attain some in the future. They have the same as the Skrell. 
Effort to get through Shields. They have a large ship, with windows, multiple entry points, expansive hallways and rooms. 
</details>

Karlious commented:
Throwing my 2c in here - I think that the ship is well designed, fits the scrap/salvage lore/flavor of the Vox perfectly.

In terms of "balance"
It uses normal engines, has no shield generator or magic box for infinite power, this ship is basically an oversized Charon, and has no base to return to.

Having thick walls is fine, and I'm sure from @bigfunguss it was more of an aesthetic choice, to give the ship a scrappy/blocky appearance, not because 3 walls = hard to assault.

bigfunguss committed:
Ship Tweaks

Fixes some oddly placed air alarms and lights, and adds them to rooms missing them. Interior walls changed from hull to standard steel, except around critical components; i.e. the gas chambers. Shuttle moved a bit further out to not be merged into the side of the ship while docked.

bigfunguss committed:
Ship tweaks round 2

Fixed interior walls being the wrong material, added some room to medbay along with an autocompressor, defibrillator, and roller bed. Moved the main shuttle air alarm as it was overlapping with the one in the airlock. Removed holopads for now since they don't quite fit with the vox theme.

bigfunguss commented:
closing this PR until I finish some necessary changes.
